src/components/TodoCard/TodoCard.tsx

```diff
@@ -0,0 +1,37 @@
+import { Todo } from '../../types/Todo';
+
+interface TodoCardProps {
+  todo: Todo
+}
+
+export const TodoCard: React.FC<TodoCardProps> = ({ todo }) => {
+  return (
+    <div data-cy="Todo" className={`todo ${todo.completed ? 'completed' : ''} `}>
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          checked={!!todo.completed}
```

Comment:
Maybe I am missing something but why not just
`checked={todo.completed}` ?

Reply:
correct